pkg/webui/console/store/selectors/application-server.js

```diff
@@ -45,3 +45,28 @@ export const selectWebhooksHealthStatusEnabled = state => {
     'unhealthy_attempts_threshold' in webhooksConfig
   )
 }
+
+export const selectIsWebhookUnhealthy = state => {
```

Comment:
How is this different from `selectWebhooksHealthStatusEnabled` ?

Reply:
this condition
`webhooksConfig.unhealthy_retry_interval !== '0s' || 'unhealthy_attempts_threshold' in webhooksConfig`
means that `selectWebhooksHealthStatusEnabled` will be true if just one `unhealthy_retry_interval` or `unhealthy_attempts_threshold` or both are not 0. There are 3 different scenarios in which the variable is true.
However, the warning is supposed to be visible in only one scenario - if BOTH are not 0. Which means I cannot use `selectWebhooksHealthStatusEnabled`. 
I didn't change the condition itself, because it is used in other places to show the health status and as I understood, it should be as it is for those cases.

Reply:
I missed the `||` vs `&&`, and it makes sense.